packages/components/src/Table/Cell/DraggableComponent.js

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import { DragSource as dragSource, DropTarget as dropTarget } from 'react-dnd';
+import PropTypes from 'prop-types';
+import flow from 'lodash/flow';
+
+/*
+ * This constant allows to specify which drag sources and drop targets are compatible.
+ */
+export const DRAGGABLE_ELEMENT_TYPE = 'element';
+
+/**
+ * This function adds the 'drag & drop' behaviour on the given component.
+ * Its returns a new component which encaspulates the given one.
+ * The new component is draggable and dropppable.
+ */
+export default function getDraggable(Comp) {
```

Comment:
I guess this is a HOC to wrap the celle renderers to inject DnD in it.
2 possibilities : 
* host it in Table/Cell but integrate it through the Table component api. I mean, accept a props.onMove, when present, wrap the renderer with this HOC.
* host it in src/, making it a component on its own, that manage DnD.

I would say number 2